bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tcanvas.py

```diff
@@ -0,0 +1,83 @@
+from . import pythonization
+
```

Comment:
We need the usual copyright header here and I would also add some documentation that will end up in the Python Interface section of the docs https://root.cern.ch/doc/master/group__Pythonizations.html. Take for example https://github.com/root-project/root/blob/master/bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tfile.py

Reply:
@vepadulano 

I apply your changes and rebase commits

Reply:
We are still missing the documentation as I suggested, see the `\pythondoc` block in `_tfile.py`. I think it should describe the fact that the `Update` method has been pythonized and a simple example usage.